src/js/media-preview-thumbnail.js

```diff
@@ -89,19 +97,45 @@ class MediaPreviewThumbnail extends window.HTMLElement {
     }
   }
 
+  /**
+   * @type {string | undefined} The url of the preview image
+   */
+  get mediaPreviewImage() {
+    return getStringAttr(this, MediaUIAttributes.MEDIA_PREVIEW_IMAGE);
+  }
+
+  set mediaPreviewImage(value) {
+    setStringAttr(this, MediaUIAttributes.MEDIA_PREVIEW_IMAGE, value);
+  }
+
+  /**
+   * @type {Array<number> | undefined} Fixed length array [x, y, width, height] or undefined
```

Comment:
nitpick(non-blocking): This should probably be `[number, number, number, number] | undefined` since it's a tuple

Reply:
I initially did this and typescript didn't like the return type because I could be technically returning an array of any length. Although I guess I could add a type comment to the return to force it?